src/Service/SF1601/DigitalPoster.php

```diff
@@ -67,17 +92,26 @@ public function sendDigitalPost(DigitalPost $digitalPost, DigitalPost\Recipient
                 MeMoHelper::SENDER_LABEL => $this->options['sf1601']['sender_label'],
             ];
 
-            $meMoMessage = $this->meMoHelper->createMeMoMessage($digitalPost, $recipient, $meMoOptions);
-            $messageUuid = $meMoMessage->getMessageHeader()->getMessageUUID();
+            // For performance reasons we want send as little a payload as possible. Using SF1601::TYPE_AUTOMATISK_VALG
+            // will require us to duplicate the content in the call to "kombiPostAfsend".
+            $meMoMessage = null;
+            $forsendelse = null;
+            $type = $this->canReceive(SF1601::FORESPOERG_TYPE_DIGITAL_POST, $recipient->getIdentifier() ?? '') ? SF1601::TYPE_DIGITAL_POST : SF1601::TYPE_FYSISK_POST;
 
-            $forsendelseOptions = [
-                ForsendelseHelper::FORSENDELSES_TYPE_IDENTIFIKATOR => $this->options['sf1601']['forsendelses_type_identifikator'],
-            ];
-            $forsendelse = $this->forsendelseHelper->createForsendelse($digitalPost, $recipient, $forsendelseOptions);
+            if (in_array($type, [SF1601::TYPE_DIGITAL_POST, SF1601::TYPE_AUTOMATISK_VALG], true)) {
```

Comment:
From what i gather `$type` can enver be `SF1601::TYPE_AUTOMATISK_VALG` here.

Reply:
You're absolutely right, but there's nothing wrong with future-proofing a solution to a problem.